src/react-apps/applications/dashboard/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "outDir": "compiled",
+    "module": "commonjs",
+    "target": "es5",
+    "lib": [
+      "es6",
+      "dom"
+    ],
+    "sourceMap": true,
+    "allowJs": true,
+    "jsx": "react",
+    "moduleResolution": "node",
+    "rootDir": "../",
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noImplicitAny": true,
+    "strictNullChecks": false,
+    "noUnusedLocals": true
+  },
+  "include": [
+    "./src/**/*",
+    "../**/src/**/*"
```

Comment:
Compile the other apps when compiling this app?

Reply:
updated with more specific includes, so only the relevant apps are included